doc/src/gotchas.rst

```diff
@@ -784,3 +784,65 @@ from SymPy 0.6.5:
         """
         expr = Poly.cancel(powsimp(expr))
         return powsimp(together(expr.expand()), combine='exp', deep=True)
+
+
+
+Be careful about substituting for derivatives! 
+==============================================
+
```

Comment:
This needs introductory text to tell the reader what you are going to be showing. When I read it I see the "Be careful" and wonder what it is that I am to be careful for. The first text is "It is worth looking at the raw form..." but I don't even know what the issue is yet.

Reply:
I see. I will make the necessary changes.